src/Symfony/Bundle/FrameworkBundle/CacheWarmer/ConfigBuilderCacheWarmer.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\CacheWarmer;
+
+use Psr\Log\LoggerInterface;
+use Symfony\Bundle\FrameworkBundle\Command\BuildDebugContainerTrait;
+use Symfony\Component\Config\Builder\ConfigBuilderGenerator;
+use Symfony\Component\Config\Builder\ConfigBuilderGeneratorInterface;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+use Symfony\Component\DependencyInjection\Extension\ConfigurationExtensionInterface;
+use Symfony\Component\DependencyInjection\Extension\ExtensionInterface;
+use Symfony\Component\HttpKernel\Bundle\BundleInterface;
+use Symfony\Component\HttpKernel\CacheWarmer\CacheWarmerInterface;
+use Symfony\Component\HttpKernel\KernelInterface;
+
+/**
+ * Generate all the config builders.
+ *
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class ConfigBuilderCacheWarmer implements CacheWarmerInterface
+{
+    use BuildDebugContainerTrait;
+
+    private $kernel;
+    private $logger;
+
+    public function __construct(KernelInterface $kernel, LoggerInterface $logger = null)
+    {
+        $this->kernel = $kernel;
+        $this->logger = $logger;
+    }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @return string[]
+     */
+    public function warmUp(string $cacheDir)
+    {
+        $generator = new ConfigBuilderGenerator($cacheDir);
+        $extensions = array_map(function (BundleInterface $a) {
+            return $a->getContainerExtension();
+        }, $this->kernel->getBundles());
+
+        foreach ($extensions as $extension) {
+            if (null === $extension) {
+                continue;
+            }
+
+            try {
+                $this->dumpExtension($extension, $generator);
+            } catch (\Throwable $e) {
```

Comment:
`\Exception` I think - catching `\Throwable` should always be avoided

Reply:
Thank you